go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/matttproud/golang_protobuf_extensions
+
+go 1.9
+
+require (
+	github.com/golang/protobuf v1.2.0
```

Comment:
This was primarily developed against pre-protobuf 1.0.0, and there were subsequent problem as they related to changes in the internal package layout of that library h(ttps://github.com/golang/protobuf/tags), namely around the test artifact files we dot-imported.

I suspect what you have here will be fine.